dolphinscheduler-task-plugin/dolphinscheduler-task-java/src/main/java/org/apache/dolphinscheduler/plugin/task/java/JavaParameters.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.plugin.task.java;
+
+import org.apache.dolphinscheduler.plugin.task.api.model.ResourceInfo;
+import org.apache.dolphinscheduler.plugin.task.api.parameters.AbstractParameters;
+
+import java.util.List;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+public class JavaParameters extends AbstractParameters {
+    /**
+     * origin java script
+     */
+    private String rawScript;
+
+    /**
+     * run in jar file
+     */
+    private ResourceInfo mainJar;
+
+    /**
+     * run type is JAVA or JAR
+     */
+    private String runType;
+
+    /**
+     * main method args
+     **/
+    private String mainArgs;
+
+    /**
+     * jvm args
+     **/
+    private String jvmArgs;
+
+    /**
+     * module path or class path flag
+     **/
+    private boolean isModulePath;
+
+    /**
+     * resource list
+     */
+    private List<ResourceInfo> resourceList;
+
+    /**
+     * @description:
+     * @date: 7/22/22 2:35 AM
+     * @param: []
```

Comment:
can we remove this unnecessary part? same as function `getResourceFilesList`

Reply:
I find it still in class `JavaTask.java`, could you search global and remove them?

Reply:
> can we remove this unnecessary part? same as function `getResourceFilesList`

this part still left cause extend the AbstractParameter, I not sure can delete

Reply:
@zhongjiajie might mean to remove the meaningless comments. @106umao  I comment in other places where the comments are meaningless. Please remove them or reword them to more meaningful comments